wear/src/main/java/io/homeassistant/companion/android/util/ToggleSwitch.kt

```diff
@@ -0,0 +1,17 @@
+package io.homeassistant.companion.android.util
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.wear.compose.material3.Switch
+import io.homeassistant.companion.android.theme.getSwitchColors
+
+@Composable
+fun getToggleSwitch(isChecked: Boolean, description: String) = Switch(

```

Comment:
This should be called `ToggleSwitch` or something like `WearSwitch` (it is a variation on `Switch` just like they do internally). If this is the only place 'our' switch colors are used it could be specified here again instead of breaking it out in a function.

Reply:
file has been renamed as we now also have checkboxes in here